.docs/content/getting-started/execution/how-to-launch-Client-Worker-Sample.md

```diff
@@ -0,0 +1,81 @@
+
+# **Testing a Sample via C# Extensions and ArmoniK Deployment**
+
+
+## Prerequisites
+
+1. **Development Environment**
+   Ensure you have a C# development environment set up with the .NET SDK installed.
+
+2. **ArmoniK Access**
+   You must have access to an ArmoniK deployment and know the path to its shared folder. [infrastructure/quick-deploy/localhost/data](https://github.com/aneoconsulting/ArmoniK/tree/main/infrastructure/quick-deploy/localhost/data)
+
+3. **C# Extensions**
+   Verify that the required C# extensions are available and correctly tagged.
+
+---
+
+## Steps to Test a Sample
+
+### Step 1: Point to the Correct Version of the C# Extensions
+
+1. **Identify the Tag**
+   Find the tag produced by your CI/CD pipeline of the C# extensions you want to use.
```

Comment:
Only work for members with permission to access the pipeline. Others cannot see pipeline related logs including the tag. You should rework your point to take that into consideration.